oci/oci.go

```diff
@@ -678,3 +680,23 @@ func (r *Runtime) RuntimeReady() (bool, error) {
 func (r *Runtime) NetworkReady() (bool, error) {
 	return true, nil
 }
+
+// PauseContainer pauses a container.
+func (r *Runtime) PauseContainer(c *Container) error {
+	c.opLock.Lock()
+	defer c.opLock.Unlock()
+	if _, err := utils.ExecCmd(r.Path(c), "pause", c.id); err != nil {
```

Comment:
remove the if and the return nil below.

Reply:
I'm not 100% sure what you're suggesting.  Is it this:
        err := utils.ExecCmd(r.Path(c), "pause", c.id)
        if err != nil {
                return err
        }
        return

Reply:
Could simplify that to
`_, err := utils.ExecCmd(...); return err`
I suppose?

Reply:
yeah most look like that ... take a peek in the same file at some of the similar functions.

Reply:
Maybe it's just me, but I think it's cleaner/clearer as is.  But if there's a strong push I can change it.  @baude thoughts?

Reply:
Oops, asked my question to @baude before seeing your reply, will look again.

Reply:
and then you just do 'return err' and that gets you nil or the error.  OK, I'm buying into it.  Thanks for the pointer and discussion.  Fixed.